.github/workflows/test.yml

```
@@ -76,6 +78,10 @@ jobs:
         echo "*** ${{ matrix.tool }} compose version ***"
         ${{ matrix.tool }} compose version
 
+    - name: Configure DOCKER_HOST for Podman
+      if: ${{ matrix.tool == 'podman' }}
+      run: echo "DOCKER_HOST=unix:///run/user/1001/podman/podman.sock" >> $GITHUB_ENV
```

```suggestion
      run: echo "DOCKER_HOST=unix://$(podman info --format '{{.Host.RemoteSocket.Path}}')" >> $GITHUB_ENV
```

Instead of hard-coding the user id, how about something like this? (Not tested, that said :))

---

Looks like it works https://github.com/Jdubrick/rhdh-local/actions/runs/18750438685